applications/FluidDynamicsApplication/tests/two_fluid_hydrostatic_pool_test.py

```diff
@@ -0,0 +1,238 @@
+    have_external_solvers = True
+except ImportError as e:
+    have_external_solvers = False
+
+import sys
+import time
+import os
+
+import KratosMultiphysics.KratosUnittest as UnitTest
+
+# Class to navigate through the folders
+class WorkFolderScope:
+    def __init__(self, work_folder):
+        self.currentPath = os.getcwd()
+        self.scope = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)),work_folder))
+
+    def __enter__(self):
+        os.chdir(self.scope)
+
+    def __exit__(self, type, value, traceback):
+        os.chdir(self.currentPath)
+
+
+# Class derived from the UnitTest (KratosMultiphysics.KratosUnittest) class
+class TwoFluidHydrostaticPoolTest(UnitTest.TestCase):
+
+    def __init__(self):
+        self.waterLevel = 0.5
+        self.work_folder = "TwoFluidStaticPoolTest"
+        self.settings = "TwoFluidStaticPoolTest2D.json"
+        self.check_tolerance = 1e-10
+        self.check_toleranceDistance = 0.03
+        self.gravitationalAcceleration = 9.81
+        self.domainHeight = 1.0
+        self.rho1 = 1000.0
+        self.rho2 = 1.0
+
+    # runs the two dimensinal test case
+    def runTwoFluidHydrostaticTest2D(self):
+        with open("TwoFluidStaticPoolTest/TwoFluidStaticPoolTest2D.json",'r') as parameter_file:
+            parameters = KratosMultiphysics.Parameters(parameter_file.read())
+            model = KratosMultiphysics.Model()
+            # running
+            self.simulation = FluidDynamicsAnalysisWithFlush2D(model,parameters)
+            self.simulation.Run()
+
+            # testing
+            for node in self.simulation._GetSolver().GetComputingModelPart().Nodes:
+                velocity = node.GetSolutionStepValue(KratosMultiphysics.VELOCITY)
+                self.assertAlmostEqual(0.0, velocity[0], delta = self.check_tolerance)
+                self.assertAlmostEqual(0.0, velocity[1], delta = self.check_tolerance)
+                self.assertAlmostEqual(0.0, velocity[2], delta = self.check_tolerance)
+
+                pressure = node.GetSolutionStepValue(KratosMultiphysics.PRESSURE)
+                if node.Y > self.waterLevel:
+                    pressureAnalytic = (self.domainHeight-node.Y)*self.gravitationalAcceleration*self.rho2
+                else:
+                    pressureAnalytic = (self.domainHeight-self.waterLevel)*self.gravitationalAcceleration*self.rho2
+                    pressureAnalytic += (self.waterLevel-node.Y)*self.gravitationalAcceleration*self.rho1
+                self.assertAlmostEqual(pressureAnalytic, pressure, delta = self.check_tolerance)
+
+                distance = node.GetSolutionStepValue(KratosMultiphysics.DISTANCE)
+                distanceAnalytic = (node.Y - self.waterLevel)
+                self.assertAlmostEqual(distanceAnalytic, distance, delta = self.check_toleranceDistance)
+            
+            # cleaning
+            try:
+                os.remove('TwoFluidStaticPoolTest2D.post.bin')
```

Comment:
please use the function from kratos_utils for this

Reply:
was done